src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html

```diff
@@ -15,7 +15,7 @@
                     <legend style="font-size: 18px;">{{tab.alias}}</legend>
 
                     <umb-property property="property"
-                                  ng-repeat="property in tab.properties">
+                                  ng-repeat="property in tab.properties | orderBy:'+sortOrder'">
```

Comment:
I didn't think performance on the sorting of a - usually short - list of fields wouldn't be an issue. But I can change to sort in the controller if you consider it might be.